src/app/components/CallToActionLinkComponent/CallToActionLinkContext.tsx

```diff
@@ -0,0 +1,9 @@
+import { createContext } from 'react';
+
+interface ContextProps {
+  size?: string;
+  fontVariant?: string;
```

Comment:
Shall we use the proper types here? 
```suggestion
  size?: FontSize;
  fontVariant?: FontVariant;
```